src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -201,6 +441,16 @@ private RequestObjectParamView generateRequestObjectParam(
     return allDocs;
   }
 
+  public SimpleParamDocView getRequestObjectParamDoc(
```

Comment:
This method and getOptionalArrayParamDoc could both return a List<ParamDocView> object like getMethodParamDocs. This simplifies the calling code and gives all the get*ParamDoc methods a common return type.